reana/cli.py

```diff
@@ -1636,23 +1657,95 @@ def client_uninstall():  # noqa: D301
     run_command("pip check")
 
 
+@click.option(
+    "-d",
+    "--debug",
+    is_flag=True,
+    default=False,
+    help="Should we build REANA in debug mode? ",
+)
 @cli.command(name="cluster-build")
-def cluster_build():  # noqa: D301
+def cluster_build(debug):  # noqa: D301
     """Build REANA cluster."""
-    for cmd in [
-        "reana-dev git-submodule --update",
-        "reana-dev docker-build --exclude-components r-ui,r-a-vomsproxy",
-    ]:
+    cmds = ["reana-dev git-submodule --update"]
+    if debug:
+        cmds.append("reana-dev python-install-eggs")
+    cmds.append(
+        "reana-dev docker-build --exclude-components r-ui,r-a-vomsproxy" + " -b DEBUG=1"
+        if debug
+        else ""
+    )
+    for cmd in cmds:
         run_command(cmd, "reana")
 
 
 @cli.command(name="cluster-deploy")
-def cluster_deploy():  # noqa: D301
+@click.option(
+    "--namespace", "-n", default="default", help="Kubernetes namespace [default]"
+)
+@click.option(
+    "-j",
+    "--job-mounts",
+    multiple=True,
+    callback=volume_mounts_to_list,
+    help="Which directories from the Kubernetes nodes to mount inside the job pods? kubernetes_node_path=inside_job_mountpath",
```

Comment:
- `cluster_node_path:job_pod_mountpath`

- perhaps we should give concrete examples using ` /var/reana/mydata:/mydata` as in reana-commons?